ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/SFLOW_TESTS_LEAFS.yml

```diff
@@ -14,7 +14,7 @@ l3leaf:
     mlag_peer_l3_ipv4_pool: 10.254.1.96/27
     spanning_tree_priority: 4096
     spanning_tree_mode: mstp
-
+    virtual_router_mac_address: 00:11:22:33:44:55
```

Comment:
not sure I understand this change?

Reply:
Because I add inband management using `inband_mgmt_subnet`, it will configure VARP, which in turn requires a vMAC.